lib/logger.js

```diff
@@ -1,6 +1,5 @@
 /*
- * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2012-2022 Digital Bazaar, Inc. All rights reserved.
  */
-const bedrock = require('bedrock');
-
-module.exports = bedrock.loggers.get('app').child('redis');
+import * as bedrock from '@bedrock/core';
+export default bedrock.loggers.get('app').child('redis');
```

Comment:
No `default` export in new / updated libs, this should be:

```suggestion
export const logger = bedrock.loggers.get('app').child('redis');
```